src/Raven.Server/Documents/ComputeHttpEtags.cs

```diff
@@ -120,6 +120,13 @@ public static unsafe string ComputeEtagForRevisions(List<Document> revisions)
             return FinalizeHash(size, state);
         }
 
+        public static unsafe string ComputeHashForTimeSeriesResult(DateTime from, DateTime to, UIntPtr size, byte* state)
+        {
+            HashNumber(state, from.GetHashCode());
```

Comment:
```suggestion
            HashNumber(state, from.Ticks);
```

We already have support for hashing long, so let's use that instead